marketplace/commercelayer-sku/src/index.js

```diff
@@ -0,0 +1,167 @@
+import { init } from 'contentful-ui-extensions-sdk';
+import debounce from 'lodash/debounce';
+import find from 'lodash/find';
+import axios from 'axios';
+import Vue from 'vue/dist/vue.esm.js';
+import './index.css';
+import 'contentful-ui-extensions-sdk/dist/cf-extension.css';
```

Comment:
Instead of importing this styles as a dependency, please, copy them only those that are used to `index.css`. It is a legacy way of doing this so we don't want to promote it.

Reply:
Should be resolved. Let me know if you need any additional changes.